docs/docs/conceptual/choosing-a-cms.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Choosing Your CMS(s)
+---
+
+At their core, most medium-sized and large Gatsby websites are backed by a so-called "headless CMS". Headless CMSes excel at allowing content editors to flexibly create a content schema of interrelated content models.
+
+One of the key questions people ask when building a Gatsby site is "which CMS should I use?"
+
+## Choosing a Headless CMS
+
+If you're looking for a primary CMS (general-purpose, flexible content modelling) to power your website, there are three main factors you can use to narrow your search:
+
+### 1) Popular, first-class Gatsby integrations
+
+**First**, when possible, we recommend you focus on **top-flight general-purpose CMSs** with **first-class Gatsby integrations** and **high popularity** among Gatsby users. A ["first-class integration"](https://support.gatsbyjs.com/hc/en-us/articles/360052503494-Developing-a-first-class-CMS-integration-for-Gatsby-Cloud) means it supports the main Cloud functionality of Gatsby -- previews and incremental builds.
+
+In terms of popularity, you can see [top integrations listed by monthly downloads here](https://www.gatsbyjs.com/plugins?=gatsby-source). There are currently (January 2021) eight CMSs used by over 1% of Gatsby users. These are:
+
+- **five general-purpose headless CMSs:** Contentful, Sanity, DatoCMS, Strapi, and Prismic
+
+- **two general-purpose full-stack CMSs** running in "headless" mode: Wordpress and Drupal
+
+- **one specialized CMS**: Shopify
+
+### 2) The right price point
+
+**Second**, it's important to consider the price point given your budget:
+
+- If you're working on a personal project or prototype, a few of these have a generous free tier (Contentful, Sanity, DatoCMS, Strapi, Prismic).
+
+- If you're looking more at "Team", "Pro", or "Business" price points (eg $29, $99, or \$299 per month), the above CMSs are all good options, plus Wordpress and Drupal .
+
+- If you're looking more at an "enterprise" project in the four digits per month and up, Contentful is the leader, with others using Contentstack, Sanity, and Strapi.
```

Comment:
I don't like the leader part. What are they leader of? Integration? UI on the dashboard?

Reply:
Good call -- lazy wording. Removed.